make_snapshots.sh

```diff
@@ -24,6 +24,11 @@ done
 if [ "${make_snapshots}" = true ] ; then
     mkdir tmp
     for filename in notebooks/*.ipynb; do
```

Comment:
in our current structure we have the new FDB/ and Polytope/ folders within notebooks/
```suggestion
    for filename in notebooks/**/*.ipynb; do
```